File: content/snippets/go/interview_lcci_16_09_operations.go
package main

import "fmt"

type Operations struct{}

func (Operations) Minus(a int, b int) int {
	return a - b
}

func (Operations) Multiply(a int, b int) int {
	return a * b
}

func (Operations) Divide(a int, b int) int {
	return a / b
}

func main() {
	ops := Operations{}
	fmt.Println(ops.Minus(10, 3))
	fmt.Println(ops.Multiply(6, -4))
	fmt.Println(ops.Divide(20, 5))
}

File: content/snippets/python/interview_lcci_16_09_operations.py
class Operations:
    def minus(self, a: int, b: int) -> int:
        return a - b

    def multiply(self, a: int, b: int) -> int:
        return a * b

    def divide(self, a: int, b: int) -> int:
        return int(a / b)


def main() -> None:
    ops = Operations()
    print(ops.minus(10, 3))
    print(ops.multiply(6, -4))
    print(ops.divide(20, 5))


if __name__ == "__main__":
    main()
